jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -278,7 +279,7 @@ public String getJarPluginName() {
 
   @Override
   public boolean isWarProject() {
-    return TaskCommon.getWarTask(project) != null;
+    return project.getPlugins().hasPlugin(WarPlugin.class);
```

Comment:
I'm guessing yes, but will this work with bootWar too?

Reply:
Yes. In fact the `bootWar` task is only created by the spring boot gradle plugin, if the `WarPlugin` is applied to the project.

Reply:
Hmm. Sorry, could you expand the import?